API/Hardware_API.md

```diff
@@ -68,6 +68,61 @@ If you're newer to hardware and these functions look like alphabet soup to you,
 
 By default, all of the pins are pulled high if not specifically set.
 
+### Pullup and Pulldown pins
+
+Pins 2-7 on both Ports are available for pullup and pulldown.
+
+The basic function of a pull-up resistor is to insure that given no other input, a circuit assumes a default value. The 'pullup' mode pulls the line high and the 'pulldown' mode pulls it low.
```

Comment:
Agreed, this is a lovely description. Can you add "pullup" and "pulldown" to the docs glossary?